routes/web.php

```diff
@@ -12,5 +12,5 @@
 */
 
 Route::get('/', function () {
-    return view('welcome');
+    return view('layout.welcome');
```

Comment:
Lets create a new file at '/resources/views/landing.blade.php`. Use that file at the route.